chapter_causality/causality.Rmd

```diff
@@ -295,155 +629,333 @@ layout: true
+From the estimation output we get the following:
 
-* It turns out that `Z` influences `Y.without.X`, which at the same time will be part of our estimator for the true effect size.
+$\begin{align} \mathbb{E}[\textrm{math score} | D_i = 0]&= \mathbb{E}[b_0 + \delta D_i + e_i | D_i = 0] \\ &= b_0 + \delta \mathbb{E}[D_i| D_i = 0] \\ &= b_0 \end{align}$
 
-* But `Z` also determines who gets treatment! All the ones with `Z > 0.9`!
+--
 
-* So, people with `Z > 0.9` get treated, but **also** have a relatively high `Y.without.X`. 
+$\begin{align} \mathbb{E}[\textrm{math score} | D_i = 1]&= \mathbb{E}[b_0 + \delta D_i + e_i | D_i = 1] \\ &= b_0 + \delta \mathbb{E}[D_i| D_i = 1] \\ &= b_0 + \delta \end{align}$
 
-* You can see that there is a depenence between the assignment and the effect size.
+--
 
+$\begin{align} ATE &= \mathbb{E}[\textrm{math score} | D_i = 1] - \mathbb{E}[\textrm{math score} | D_i = 0] \\ &= b_0 + \delta - b_0 \\ &= \delta \end{align}$
+
+--
+
+We knew this already but we now understand why this is true `r emo::ji("v")`
 
 ---
 
-class: title-slide-final, middle
+class:inverse
 
-# THANKS
+# Task 4 (10 minutes)
 
-To [Nick Huntington Klein](http://www.nickchk.com/index.html) for making his lecture material available online!
+1. Filter the dataset to only keep first graders and the small class and regular class groups.
+
+1. Compute the average math score for both groups, and the difference between the two. (Use base `R`.)
+
+1. Create a dummy variable `treatment` equal to `TRUE` if student is in treatment group (i.e. small class size) and `FALSE` if in control group (i.e. regular class size). See slide 33 for how to create a dummy variable.
+
+1. Regress the treatment dummy variable on math score. Are the results in line with the previous question?
+
+1. How do you interpret these coefficients?
+
+---
+
+# Shortcomings of RCTs 
+
+RCTs have very strong ***internal validity***, that is they can convincingly establish causal links.
+
+However, they have some shortcomings:
+
+* RCT are often **infeasible**:
+
+  * RCTs are **costly**,
+  * RCTs may face some **ethical issues**: some *treatments* simply cannot be given to people,
+  * RCTs take time and we may be **time constrained**.
+
+--
+
+* **Interpretation** of the results:
+
+  * ***External validity***: To what extent can the results from a given RCT be generalized to other contexts (countries, populations,...)?
+  
+  * Uncovering the mechanisms that are at stake may be difficult,
+  
+  * Imperfect randomization, attrition, ...
+
+---
+
+
+# What comes next? 
+
+* So if we cannot rely on RCTs to make our life easy, it means we have to find a way to make causal inference from ***observational data*** (as opposed to ***experimental data***).
+
+--
+
+* It brings us back to models 
+
+  - In causal inference, the *model* is our idea of what the process that *generated the data* is.
+
+  - We have to make some assumptions about what this is!
+
```

Comment:
note: even though it's often disregarded, we make pretty strong assumptions when setting up an RCT as well. Additive linearity of the PO model is *not* innocuous, for example. I like the transition to observational data, but maybe that point could be incorporated in the next iteration.